src/styles/blocks/_calendar.scss

```
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: $calendar-size;
+  margin: 0 auto;
+  padding: 20px 0;
+  gap: $cell-gap;
+  &__cell {
+    position: relative;
+    @include square($cell-size);
+    font-size: 30px;
+    font-family: Arial, Helvetica, sans-serif;
+    border: 1px solid #000;
```

create vars for colors

```suggestion
    border: 1px solid $c-black;
```